apps/backend/src/db/migrations/20240906080253_add_allow_notifications_column_to_user_details.ts

```diff
@@ -0,0 +1,19 @@
+import { type Knex } from "knex";
+
+const TABLE_NAME = "user_details";
```

Comment:
@fshabanov  Can we use the database-table-name enum here or will it interfere with how migration works?

Reply:
It will break the migration. We move tables names only to local migration constants